18F/ux-guide
# Redirects for pages appearing empty from search results

Closes #245 
Closes #230 

[Preview](https://federalist-48b3228b-0241-4802-9442-e08ff5c3e680.app.cloud.gov/preview/18f/ux-guide/245-fix-broken-pages-display/)

The UX Guide has resources that currently display as markdown files in GitHub.
When search results send people to these pages, they appear to have no content because of the layout being used. While the UX team decides how these pages should in fact display, this PR fixes the issue by redirecting to the GitHub pages. (These are the links used in the site for these resources. See https://ux-guide.18f.gov/resources/ )

To test: 
1. Visit the resources links on the [Resources page](https://ux-guide.18f.gov/resources/)
2. On the GitHub page where you arrive from the link in step 1, copy the permalink at the top of the content
3. Back on the preview of the Guide, append the permalink text you copied to the site's home page url
Expected behavior: You should be redirected to the GitHub page
